freqtrade/freqai/data_kitchen.py

```diff
@@ -1247,17 +1247,21 @@ def populate_features(self, dataframe: DataFrame, pair: str, strategy: IStrategy
         tfs: List[str] = self.freqai_config["feature_parameters"].get("include_timeframes")
 
         for tf in tfs:
+            metadata = {"pair": pair, "tf": tf}
             informative_df = self.get_pair_data_for_features(
                 pair, tf, strategy, corr_dataframes, base_dataframes, is_corr_pairs)
             informative_copy = informative_df.copy()
 
             for t in self.freqai_config["feature_parameters"]["indicator_periods_candles"]:
+                metadata["period"] = t
```

Comment:
The `period` is already sent as a `t` parameter. Do you thinks it's necessary to add in metadata?

Reply:
Good point, I removed it to avoid redundancy.